deploy/kubernetes/elastic-agent-standalone-kubernetes.yml

```diff
@@ -87,6 +111,137 @@ data:
       node: ${NODE_NAME}
       scope: node
     inputs:
+      - id: 4ae27079-6cd4-4ab7-a459-abbae74ffc44
```

Comment:
You should be able to remove the `id` from everywhere. You really do not need those, being that those are normally generated by Fleet.

Elastic Agent will work without `id` on the inputs and streams.

Reply:
👍🏼 thanks for clarifying